pymc3/plots.py

```diff
@@ -779,12 +779,12 @@ def get_trace_dict(tr, varnames):
         if ax is None:
             ax, fig = create_axes_grid(figsize, trace_dict)
 
-        for a, v in zip(ax, trace_dict):
+        for a, v in zip(np.atleast_1d(ax), trace_dict):
```

Comment:
I'm not trying to flatten, I'm trying to ensure that a single axis passed is iterable.